src/index.ts

```diff
@@ -0,0 +1,234 @@
+import {
+  absCal,
+  addTheValueToMemory,
+  buttonVisibilityHandler,
+  calculationOfSimpleCal,
+  celiNumberCal,
+  changeButtonColor,
+  changeInUnitOfAngle,
+  changeTheValue,
+  degToDms,
+  drawerClose,
+  drawerShow,
+  dynamicStyleDrawer,
+  dynamicStyleDrawerWithDebounce,
+  floorNumberCal,
+  getValueFromLocal,
+  inputField,
+  inputToDeg,
+  isOperationPresent,
+  isTrigonoCal,
+  powerAndRootCal,
+  randomNumberGenerator,
+  recallTheValueFromMemory,
+  removeCharFromCal,
+  removeNumbers,
+  removeTheValueFromMemory,
+  secondBtnShow,
+  setCharAtInputField,
+  setValueInLocal,
+  showStoredNumbers,
+  simpleCalculation,
+  stringPreAdder,
+  toExponentialConvert,
+} from "./functions";
+let flagForToggleBtn: boolean = false;
+let flagForHypBtn: boolean = false;
```

Comment:
`boolean` will be inferred

Reply:
Please add an empty line after import statements for better readability.

Reply:
removed the boolean and added the line for the better readability.